Source/WTF/wtf/win/MemoryFootprintWin.cpp

```diff
@@ -44,7 +44,7 @@ size_t memoryFootprint()
     // > The shared data includes pages that contain all instructions your application executes,
     // > including those in your DLLs and the system DLLs. As the working set size increases,
     // > memory demand increases.
-    Win32Handle process(OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, GetCurrentProcessId()));
```

Comment:
I don't like this line contains the word `Win32Handle` twice. I'd like to write
~~~c++
auto process = Win32Handle::adopt(OpenProcess(PROCESS_QUERY_INFORMATION, FALSE, GetCurrentProcessId()));
~~~
WDYT?

Reply:
I agree that it’s more verbose than I’d like but I wanted to keep with the UnixFileDescriptor constructs since we have to touch those when working on PlayStation and having a common language is good.

Reply:
It's not WebKit's common style. 
For example, GDIObject class has `adoptGDIObject`.
https://github.com/WebKit/WebKit/blob/4c8182f7f87474c644e5cf5eaf01207778b150ff/Source/WTF/wtf/win/GDIObject.h#L105

Reply:
Ok added an additional convenience method to construct.